.github/workflows/convert-yaml-to-json.yml

```diff
@@ -0,0 +1,26 @@
+name: convert-yaml-to-json
+
+on:
+  [push]
```

Comment:
Question, where will the pushed yaml live?

we should run this action only when we see yaml being pushed up, so maybe we can have a yaml/ folder at root so people push their files there. Then we can have this action run against changes on that folder like so:
```
on:
  push:
    paths:
      - 'yaml/**.yaml'

```
see [this page](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) for more details

Reply:
I think that's a great idea!

(I don't think we had decided on a folder for the pushed yaml, so `yaml` in the root is :+1: )